tests/test_api.py

```diff
@@ -695,6 +740,38 @@ def test_is_delegated_role(self) -> None:
             self.assertFalse(role.is_delegated_path("a/non-matching path"))
             self.assertTrue(role.is_delegated_path("a/path"))
 
+    def test_zero_padding_in_succinct_roles(self) -> None:
+        bit_lengths = [2, 6, 8, 10, 16, 32]
+        expected_paddings = [1, 2, 2, 3, 4, 8]
+
+        for bit_length, expected in zip(bit_lengths, expected_paddings):
+            succinct_delegation = SuccinctRoles([], 1, bit_length, "foo")
+            m = f"Error for {bit_length} expected padding {expected}"
+            self.assertEqual(succinct_delegation.suffix_len, expected, m)
+
+    def test_is_delegated_role_in_succinct_roles(self) -> None:
+        succinct_roles = SuccinctRoles([], 1, 5, "bin")
+        # delegated role name suffixes are in hex format.
```

Comment:
"bin-20" and "bin-100" are valid hex numbers, so the comment feels a bit odd

Reply:
Okay, will move this comment before the `true_name_examples` and then it will make more sense.